tests/unit/amo/components/TestAddonsByAuthorsCard.js

```diff
@@ -445,4 +446,26 @@ describe(__filename, () => {
     expect(root.find(AddonsCard))
       .toHaveProp('header', 'More add-ons by these developers');
   });
+
+  it('shows header text without More text', () => {
+    const root = renderAddonsWithType({
+      addonType: ADDON_TYPE_THEME,
+      showMore: false,
+      multipleAuthors: false,
+    });
+
+    expect(root.find(AddonsCard))
+      .toHaveProp('header', `Themes by ${fakeAuthor.name}`);
+  });
+
+
+  it('shows header text starting with More text as default', () => {
+    const root = renderAddonsWithType({
+      addonType: ADDON_TYPE_THEME,
+      multipleAuthors: false,
+    });
+
+    expect(root.find(AddonsCard))
+      .toHaveProp('header', `More themes by ${fakeAuthor.name}`);
+  });
```

Comment:
Is this needed since we have existing test cases?

Reply:
yep -you're right...it's a dupe test and not needed here, thanks